### ISSUE TITLE
Valheim patch 0.217.29 

### ISSUE THREAD

**Issue by Wayne-x6y** [BepInEx/BepInEx#699]
### What happened?

BepInEx 5.4.2200 not working with Valheim patch 0.217.29 (Nov 8 2023).
Valheim does not start (no error messages). If I rename BepInEx folder, it starts fine. If I copy in a new BepInEx folder with no mods, Valheim does not start. 
.\BepInEx\LogOutput.log only shows last successful load (pre-patch).
I have tried this on two different Windows 11 machines.

### BepInEx Distribution

Stable from GitHub

### Environment

```markdown
- OS: 
- BepInEx: 
- Game:
```

**Comment by schuhwerk** [comment]
Same issue, just rolled back to the version from a few days ago

Enter: ``steam://nav/console`` (via- Browser or <kbd>WIN</kbd> + <kbd>R</kbd>)
In the Steam-Console enter ``download_depot 892970 892972 7324760316484937832``
This downloads an older Valheim version (windows).

More Details:
https://steamcommunity.com/sharedfiles/filedetails/?id=889624474
https://steamdb.info/app/892970/depots/

**Comment by VulcanTourist** [comment]
This is almost certainly related to [#694](https://github.com/BepInEx/BepInEx/issues/694).

**Comment by Wayne-x6y** [comment]
Agreed due to earlier patch (issue #694). Good news is that BepInEx from Thunderstore 5.4.2202 (manual install) works with latest patches. I didn't try it since instructions said .220x were minor and didn't require reinstall. But Vapok on his discord suggested the update. I tested and it works with all my previous mods (I have about 18 of them).

**Comment by VulcanTourist** [comment]
> Agreed due to earlier patch (issue #694). Good news is that BepInEx from Thunderstore 5.4.2202 (manual install) works with latest patches. I didn't try it since instructions said .220x were minor and didn't require reinstall. But Vapok on his discord suggested the update. I tested and it works with all my previous mods (I have about 18 of them).

I had to dump the denikson "Pack" - that includes a modified BepInEx - altogether in order to get the game to load with mods.  As long as the Pack version 5.4.2202 was present, BepInEx would abort because of a Preloader (which is modified from vanilla) failure and the game would run as ***vanilla***.  I'm using nothing but vanilla BepInEx from Github and that is all that worked for me.

**Comment by JordanW9232** [comment]
> Agreed due to earlier patch (issue #694). Good news is that BepInEx from Thunderstore 5.4.2202 (manual install) works with latest patches. I didn't try it since instructions said .220x were minor and didn't require reinstall. But Vapok on his discord suggested the update. I tested and it works with all my previous mods (I have about 18 of them).

Omg, THANK YOU. I spent days trying to get this working on my steam deck, probably about 4-5 hours in total. After seeing this comment I got it working in like 5 minutes lol. Just needed that 5.4.2202 version.
